### PR TITLE
Fix links to ATM-fraud defaults

### LINKED ISSUE #220: Test docs generation with mkdocs
To avoid problems with publishing such as [this one](https://github.com/bakdata/kpops/actions/runs/5044422510/jobs/9047562651), we should test docs generation with mkdocs as part of the CI or a pre-commit hook.

Could be as simple as a line in the CI `just serve-docs`.

### PR DESCRIPTION
**Pull request by sujuka99**
Closes #220 
#216 updated the example pipeline file structure, but this was not reflected in the links to the docs.

See #218

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -55,6 +55,12 @@ jobs:
       - name: Test
         run: poetry run pytest tests
 
+      - name: Install docs dependencies
+        run: poetry install --with docs
```
Comment (disrupted) [inline .github/workflows/ci.yaml:59]: any reason for not adding this to the initial poetry install above?

Comment (sujuka99) [reply .github/workflows/ci.yaml:59]: to save time if anything fails fefore that

Comment (disrupted) [reply .github/workflows/ci.yaml:59]: ah, yeah that makes sense. I don't have a preference, you can decide yourself what you think is most appropriate

Comment (sujuka99) [reply .github/workflows/ci.yaml:59]: I prefer to keep it like it is currently, it seems that it is only installing the doc dependencies after all

**Review by disrupted — APPROVED** [review]

**Pull request merged**